### ISSUE TITLE
Missing credo warnings in test directory when checking  in app of umbrella project

### ISSUE THREAD

**Issue by jonnysod** [rrrene/credo#428]
### Environment

* Credo version (`mix credo -v`): 0.8.6
* Erlang/Elixir version (`elixir -v`): 1.5.1
* Operating system: macOS 10.12.6

### What were you trying to do?

Execute `mix credo --strict` in an app (`child`), that is part of an umbrella (`credotest`) project.

```
~/credotest/apps/child$ mix credo --strict
```

### Expected outcome

```
  Code Readability                                                              
┃ 
┃ [R] ↓ There should be no trailing white-space at the end of a line.
┃       apps/child/test/child_test.exs:4:1 #(ChildTest)
┃ [R] ↓ Line is too long (max is 80, was 88).
┃       apps/child/mix.exs:30:81 #(Child.Mixfile.deps)
```
when the check is executed in the umbrella project, the output is correct (`~/credotest$ mix credo --strict`).

### Actual outcome

No credo warnings

### Create `credotest` project
```
mix new credotest --umbrella
cd credotest/
sed -i.bak 's/\[\]/[{:credo, "~> 0.8", only: [:dev, :test], runtime: false}]/' mix.exs
mix deps.get
cd apps/
mix new child
cd child/
sed -i.bak '28i\
\ \ \ \ \ {:credo, "~> 0.8", only: [:dev, :test], runtime: false}
' mix.exs
mix deps.get
sed -i.bak '4i\
\ \ ' test/child_test.exs
mix credo --strict
cd ../..
mix credo --strict
```

**Comment by rrrene** [comment]
@jonnysod This does sound like a bug. Thx for reporting! :+1:

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

If you have a `.credo.exs` make sure it includes the test directory:

```elixir
        included: ["lib/", "src/", "test/", "web/", "apps/"],
```

Please report back if your issue is solved! 👍

**Comment by weisslj** [comment]
@rrrene Was this fixed by 6089bd5?

@jonnysod Can you verify this is fixed for you? For me it is since this change.

**Comment by jonnysod** [comment]
@rrrene Thanks the issue is solved for me.
